core/src/robot/FixedFrameConstraint.cc

```diff
@@ -0,0 +1,65 @@
+#include "jiminy/core/robot/Robot.h"
+#include "jiminy/core/Utilities.h"
+
+#include "jiminy/core/robot/FixedFrameConstraint.h"
+
+
+namespace jiminy
+{
+    FixedFrameConstraint::FixedFrameConstraint(std::string const & frameName) :
+    AbstractConstraint(),
+    frameName_(frameName),
+    frameId_(0)
+    {
+        // Empty on purpose
+    }
+    FixedFrameConstraint::~FixedFrameConstraint(void)
+    {
+        // Empty on purpose
+    }
+
+    hresult_t FixedFrameConstraint::initialize(Model *model)
```

Comment:
spacing

Reply:
Better use a const * instead if possible (or make it possible if it is not too much trouble)